content/en/docs/Usage/backup.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Automated Backup
+linkTitle: Automated Backup
+date: 2024-12-21
+description: >
+  Information on the in-built backup system
+---
+
+Navidrome version 0.54.x introduces a backup feature that allows the music server's data to get periodically exported and optionally re-imported. This guide will walk you through configuring backups using both the config.toml file and environment variables, where to locate the backups, and how to restore from a backup.
+
+{{% pageinfo %}}
+Note: The backup process ONLY backs up the database (users, play counts, etc.). It does NOT back up the music or the config.
+{{% /pageinfo %}}
+
+## Configuring Backup with config.toml
+
+To configure backups using the navidrome.toml file, insert the following lines to set up backups:
+
+~~~conf
+[Backup]
+Path = "/path/to/backup/folder"
+Count = 7
+Schedule =  "0 0 * * *"
+~~~
+
+* Backup.Path: The directory where backups will be stored. Replace "/path/to/backup/folder" with the desired path.
+* Backup.Count: The number of backup files to keep.
+* Backup.Schedule: `cron`-like syntax to define how often backups occur. The example above schedules a backup every 24 hours at midnight.
+
+
+## Configuring Backup with Environment Variables
+
```

Comment:
Provide note that for systemd, if not in `/var/lib/navidrome`, you may need to add the path to read write path

Reply:
Is it fair to say that that ONLY applies to linux installations? manual, rpm, and deb? Or could systemd ever need configured for containerized ND services? I'm thinking I'd like to either put the notes on linux systems that run ND through systemd as part of the "Additional Considerations" at the bottom or create a separate section dedicated to it.

Reply:
Yes, only bare metal